Dangerfile

```diff
@@ -1,5 +1,9 @@
 commit_lint.check
 
+if github.pr_title.include? "[WIP]"
```

Comment:
How about this?
```
github.pr_title.upcase.include? 'WIP'
```
This will also pick up; 
```
WIP:
WiP:
```
as well as the original, more specific, `[WIP]`